Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -344,7 +344,7 @@ public void StartOrResumeGame(bool randomize)
                 LogError($"Error invoking OnArchipelagoGameStarted:\n {ex}");
             }
 
-            HintTracker.Start(session);
+            ItemChangerMod.Modules.Add<HintTracker>();
```

Comment:
This probably belongs in the AddModules method of ArchipelagoRandomizer. Also this will be re-added on reloaded saves